components/Image.html

```diff
@@ -36,7 +36,49 @@
                 </ul>
             </aside>
             <div class="main__grid-item">
-               
+                <div class="component">
+                    <p class="component__heading">Responsive Images</p>
+                    <p class="component__text">Responsive Image Components are used to display Images 
+                        on websites, which are screen responsive. They expand upto the width of the 
+                        container they are in.</p>
+                    <p class="component__heading-primary">Round Images</p>
+                    <p class="component__sub-text">
+                         
+                    </p> 
```

Comment:
there is no content inside this paragraph. you can add something here.

Reply:
line 27 can be a heading `h3`/`h4`